### ISSUE TITLE
6th edition JetBrains Rider has a Free non-commercial version now

### ISSUE THREAD

**Issue by dogruis** [rbwhitaker/CSharpPlayersGuideEarlyAccess#723]
```
The only non-Microsoft IDE on this list is Rider from JetBrains. JetBrains makes a lot of IDEs
for other languages (PyCharm, WebStorm, IntelliJ), so while Rider is relatively new, it is still a
fantastic IDE made by people who understand IDEs. Rider works on Windows, macOS, and
Linux. Rider is not free; the cheapest option is about $150 per year.
If you have the money to spend, this is a good choice on any operating system.
```

This paragraph is not correct anymore. They introduced a version Free for non-commercial use. You should check their new licensing strategy. https://www.jetbrains.com/rider/#

**Comment by rbwhitaker** [comment]
Yeah, I saw that! It's too bad it isn't free for commercial use for small teams like Visual Studio Community edition is, but it's still free for people learning, which is really nice. I'll need to get that updated. (The free version of Rider is a very recent thing. Within the last few weeks, I think.)

Thanks for bringing it up. I'll make sure it gets addressed.

**Comment by dogruis** [comment]
I don't mind reviewing your entire book. I will go through it and post every time I see something worth mentioning. I am a backend software engineer but I am working on my own monogame content and started reading your book.

**Comment by rbwhitaker** [comment]
Accounted for now.
